src/index.html

```diff
@@ -11,7 +11,144 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="blocks">
+        <legend>Personal information</legend>
+        <label class="label">Surname:
+          <input
+            class="inputs-first"
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">Name:
+          <input class="inputs"
+          type="text"
+          name="name"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            class="inputs"
            type="text"
            name="name"
            autocomplete="off"
          >
```

Reply:
when you started moving attributed - do it for all :)
and don't forget about indentation
fix everywhere